### PR TITLE
Allows WorkflowLayout to work from any stream of renderings.

### LINKED ISSUE #394: Make ViewRegistry accept just a stream of renderings
Need to just start showing a stream of renderings that are sourced from an arbitrary observable.

cc @ellapolo

### PR DESCRIPTION
**Pull request by rjrjr**
Closes #394.

### CODE REVIEW

**Comment by zach-klippenstein** [comment]
Cc @ellapolo

File: kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowLayout.kt [hunk 1]
```diff
@@ -42,15 +42,28 @@ class WorkflowLayout(
...
+    registry: ViewRegistry
+  ) {
+    takeWhileAttached(renderings) { show(it, registry) }
+  }
+
+  /**
+   * Convenience override to start this layout from [renderings][WorkflowRunner.renderings]
+   * and [viewRegistry][WorkflowRunner.viewRegistry] of [workflowRunner].
+   */
+  fun start(workflowRunner: WorkflowRunner<*>) {
```
Comment (rjrjr) [inline kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowLayout.kt]: @zach-klippenstein  Tempted to make this an extension in WorkflowRunner.kt. WDYT?

Comment (rjrjr) [reply kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowLayout.kt]: I guess that seems backward. Better for the layout to depend upon the model than vice versa.

Comment (zach-klippenstein) [reply kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowLayout.kt]: Sounds good!

**Comment by rjrjr** [comment]
Oops, accidentally hit the re-review button. No changes, just waiting on Travis to merge.

**Pull request merged**